### PR TITLE
add zh_cn translation

### PR DESCRIPTION
**Pull request by HfSrJun**

### CODE REVIEW

**Comment by tristankechlo** [comment]
First of all, thanks for taking the time translating the mod!  
  
Now I got a few points:
 - If you want to be credited, also add your name to the Translations part of the Readme.md
 - Are you ok with me, bringing the translation to newer modversions?

File: src/main/resources/assets/additionalredstone/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+	"itemGroup.additionalredstone.main" : "更多红石元件",
```
Comment (tristankechlo) [inline src/main/resources/assets/additionalredstone/lang/zh_cn.json]: Can you reverse this line to `Additional Redstone`, I like to keep the item-group name consistent between all languages

Comment (HfSrJun) [reply src/main/resources/assets/additionalredstone/lang/zh_cn.json]: if you want, yes.
"additional redstone" is not hard to understand, so keeping the English name is ok

**Comment by HfSrJun** [comment]
> First of all, thanks for taking the time translating the mod!
> 
> Now I got a few points:
> 
> * If you want to be credited, also add your name to the Translations part of the Readme.md
> * Are you ok with me, bringing the translation to newer modversions?

feel free to use the translation in any other versions
and if anything is changed or the translation has mistakes, I will edit to correct it

**Comment by tristankechlo** [comment]
Thank you for the translation

**Pull request merged**